### PR TITLE
Add resize prop

### PR DESCRIPTION
**Pull request by jibberishclient**
I think TextAreas should have the ability to resize, in most cases it will be full width with the ability to resize vertically. I think this should be accepted as a prop rather than making the user specify it in the styles themselves.

### CODE REVIEW

**Comment by jibberishclient** [comment]
@Chrischuck Kinda new to PRs, can you take a look?

File: src/textarea/src/Textarea.js [hunk 1]
```diff
@@ -116,6 +123,9 @@ class Textarea extends PureComponent {
         {...(disabled ? { color: 'muted' } : {})}
         {...Textarea.styles}
         {...props}
+        style={{
+          resize: resize      
+        }}
```
Comment (mshwery) [inline src/textarea/src/Textarea.js:128]: Ideally this should be handled by `ui-box` not Evergreen. That way it's handled anytime you use a `textarea` not just Evergreen's `<Textarea />` component.

Comment (jibberishclient) [reply src/textarea/src/Textarea.js:128]: As far as I could tell, ui-box doesn't include a resize prop

Comment (mshwery) [reply src/textarea/src/Textarea.js:128]: You are correct! I'm suggesting we add it there.

Comment (jibberishclient) [reply src/textarea/src/Textarea.js:128]: Ya for sure, that makes sense

**Comment by mshwery** [comment]
Closing this as we should target `ui-box` for supporting `resize` as a css prop.

**Comment by mshwery** [comment]
@jibberishclient you should be to upgrade ui-box directly if you want now! 
https://github.com/segmentio/ui-box/releases/tag/v3.3.0

Will work on upgrading in `evergreen-ui` eventually :)

**Pull request closed without merging**